pkg/query-service/app/dashboards/model.go

```diff
@@ -559,20 +582,33 @@ func countPanelsInDashboard(data map[string]interface{}) model.DashboardsInfo {
 												metricsPanelCount++
 											} else if data["dataSource"] == "logs" {
 												logsPanelCount++
+												logsPanelsWithAttrContains += checkLogPanelAttrContains(data)
 											}
 										}
 									}
 								}
 							}
+						} else if ok && query["queryType"] == "clickhouse_sql" && query["clickhouse_sql"] != nil {
+							if isDashboardWithLogsClickhouseQuery(inputData) {
+								logChQuery = true
+							}
 						}
 					}
 				}
 			}
 		}
 	}
+
+	logChQueryCount := 0
+	if logChQuery {
+		logChQueryCount = 1
+	}
```

Comment:
What is happening here?

Reply:
this is just telling that this dashboard has atleast one panel with clickhouse query.